### PR TITLE
Add yoruba text c3

### PR DESCRIPTION
**Pull request by dadelani**
Added Yoruba texts for training embeddings and language models based on the paper https://www.aclweb.org/anthology/2020.lrec-1.335/

### CODE REVIEW

**Comment by lhoestq** [comment]
looks like this PR includes changes about other datasets

Can you create another branch and another PR to only include the files of yoruba text c3 please ?

File: datasets/yoruba_text_c3/yoruba_text_c3.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for BookCorpus.
+        Args:
+        **kwargs: keyword arguments forwarded to super.
+        """
+        super(YorubaTextC3Config, self).__init__(**kwargs)
+
+
+class YorubaTwiTextC3(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/yoruba_text_c3/yoruba_text_c3.py:62]:
```suggestion
class YorubaTextC3(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `yoruba_text_c3`

you'll have to regenerate the dataset_infos.json file after this change

**Review by lhoestq — COMMENTED** [review]
Thanks for adding this one :) 

I left a comment

can you also recreate the PR to only include the files of yoruba text c3 pelase ?

**Comment by dadelani** [comment]
Thanks for the review. I'm a bit confused how to remove the files. Every time I add a new branch name using the following commands:

git fetch upstream
git rebase upstream/master
git checkout -b a-descriptive-name-for-my-changes

and push to the origin, this issue occurs

**Comment by lhoestq** [comment]
Can you try to create the branch from the master branch of your fork ?

first update your master branch:
```
git checkout master
git fetch upstream
git rebase upstream/master
git push
```

then create a new one:
```
git checkout -b my-new-branch
```

**Comment by lhoestq** [comment]
I think you were still having the files because you were creating the new branch from a branch in which you've committed the files, instead of creating the new branch from the master branch

**Comment by dadelani** [comment]
Got it, will correct that. Thanks

**Comment by dadelani** [comment]
@lhoestq , I have removed the unnecessary files. Looks like I still have one error. How do I resolve this?

**Comment by cstorm125** [comment]
> @lhoestq , I have removed the unnecessary files. Looks like I still have one error. How do I resolve this?

I think it's connection error on piqa dataset. Can you try triggering the test again? I usually resolve similar issues with:
```
git fetch upstream
git rebase upstream/master
git push -u -f origin your_branch_name
```

**Comment by dadelani** [comment]
thank you  @cstorm125

**Comment by dadelani** [comment]
I have created another pull request for this https://github.com/huggingface/datasets/pull/1515 @cstorm125  @lhoestq

**Comment by ijaz314** [comment]
Hii please follow me

**Review by lhoestq — APPROVED** [review]
Looks all good now thanks !

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Comment by dadelani** [comment]
Great, thanks a lot

**Pull request merged**